react/useFullSession.tsx

```diff
@@ -0,0 +1,21 @@
+import { QueryHookOptions, useQuery } from 'react-apollo'
+
+import type { Session } from './SessionTypes'
+import SessionQuery from './graphql/session.graphql'
+
+interface Data {
+  session: Session
+}
+
+interface Variables {
+  items?: string[]
```

Comment:
Same here, can we make this safer somehow?

Reply:
It can be anything